src/parser/shared/parserGenerator.ts

```diff
@@ -56,7 +56,8 @@ export function createParser(
                 curr.code,
                 contextIndex,
             );
-            const snippet = snippetMap?.[curr.code].at(-1);
+            const snippetsForCode = snippetMap?.[curr.code]
+            const snippet = snippetsForCode?.[snippetsForCode.length - 1]
```

Comment:
For someone who wonder why `snippetsForCode.length` doesn't have optional chaining:

Since index access operator has optional chaining, it's guaranteed to have `snippetsForCode` when the inner part of `[]` is evaluated.